src/Connector/ApiKeyConnector.php

```diff
@@ -100,4 +100,19 @@ public function delete($path, $form_params = [], $query_params = [], $headers =
 
         return json_decode($response->getBody()->getContents());
     }
+
+    public function postFile($path, $multipart, $query_params = [], $headers = [])
```

Comment:
@biplobice 
I also want to add the postFile method to ConnectorInterface.php

Reply:
Added.